hushline/model.py

```diff
@@ -28,6 +28,28 @@ def default(cls) -> "SMTPEncryption":
         return cls.StartTLS
 
 
+class HostOrganization(Model):
+    __tablename__ = "host_organization"
+
+    _DEFAULT_PRIMARY_KEY: int = 1
+    _DEFAULT_BRAND_PRIMARY_HEX_COLOR: str = "#7d25c1"
+    _DEFAULT_BRAND_APP_NAME: str = "🤫 Hush Line"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    brand_app_name: Mapped[str] = mapped_column(db.String(255), default=_DEFAULT_BRAND_APP_NAME)
```

Comment:
I'm not sure how Alembic interprets `default` args to `mapped_column`, but we might need to make sure that the generated migrations don't have a server side default on the column. I think this happened to me earlier when testing some stuff on a different branch :shrug:

Reply:
I think `default=` sets the default value when the DB returns NULL. `server_default=` configures a default value in the table definition to be returned by the DB (https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.Column.params.server_default)